tests/lightgbm/test_optimize.py

```diff
@@ -15,12 +15,13 @@
 import pytest
 
 import optuna_integration
-from optuna_integration._lightgbm_tuner.optimize import _BaseTuner
-from optuna_integration._lightgbm_tuner.optimize import _OptunaObjective
-from optuna_integration._lightgbm_tuner.optimize import _OptunaObjectiveCV
-from optuna_integration._lightgbm_tuner.optimize import LightGBMTuner
-from optuna_integration._lightgbm_tuner.optimize import LightGBMTunerCV
+import optuna_integration.lightgbm
```

Comment:
In my environment, the test worked without this line.
Since we import the `optuna_integration` module, we might not need to import the public modules under it.
```suggestion
```